src/v/redpanda/admin/api-doc/features.json

```diff
@@ -86,6 +124,43 @@
                     "items": {"type": "feature_state"}
                 }
             }
+        },
+        "license_contents" :{
+            "id": "license_contents",
+            "description": "Parameters belonging to a valid, signed redpanda license",
+            "properties": {
+                "version": {
+                    "type": "int",
+                    "description": "license schema version evolution number"
+                },
+                "org": {
+                    "type": "string",
```

Comment:
this must be a required prorperty. the validator should reject licenses that do not have the company-name associated w/ it.

Reply:
This is the type of the response the server sends back to the client. The user sends a license as part of the payload of a HTTP PUT message, there is proper validation for that.